README.md

```diff
@@ -11,7 +11,13 @@ git clone --recursive https://github.com/wsmoses/Polygeist.git
 cd Polygeist
```

Comment:
Un-related but I find this command ```cd Polygeist``` a bit confusing considering that the following commands assume to be in the parent directory of Polygeist. What do you think of dropping it?

Reply:
Makes sense to me.

Reply:
Or, I can keep it, since that is generally the step I'd take after cloning a repo, but remove the `Polygeist/` prefix from the other commands.